sphinx_immaterial/custom_admonitions.py

```diff
@@ -413,6 +486,10 @@ def setup(app: Sphinx):
     app.connect("env-check-consistency", add_admonition_and_icon_css)
     app.connect("config-inited", on_config_inited)
 
+    # override original version directives with custom derivatives
+    for name, __ in VERSION_DIR_STYLE.items():
```

Comment:
not a bug but probably you intended just one underscore there?

Reply:
Originally, yes. But `_` is actually already used as a translation function (see module imports).